docs/zh/concept/config.md

```diff
@@ -68,23 +66,21 @@ sink {
 
 ### env
 
-用于添加引擎可选的参数，不管是什么引擎（Zeta、Spark 或者 Flink），对应的可选参数应该在这里填写。
+用于配置引擎相关的可选参数。无论是使用Zeta、Spark还是Flink引擎，相应的配置参数都应该在此处设置。
 
-注意，我们按照引擎分离了参数，对于公共参数我们可以像以前一样配置。对于Flink和Spark引擎，其参数的具体配置规则可以参考[JobEnvConfig](./JobEnvConfig.md)。
+需要注意的是，参数配置已按引擎类型进行了分离。通用参数可以按原有方式配置，而Flink和Spark引擎的特定参数配置规则请参考[JobEnvConfig](./JobEnvConfig.md)。
 
 <!-- TODO add supported env parameters -->
 
 ### source
 
-source用于定义SeaTunnel在哪儿检索数据，并将检索的数据用于下一步。
-可以同时定义多个source。目前支持的source请看[Source of SeaTunnel](../connector-v2/source)。每种source都有自己特定的参数用来
-定义如何检索数据，SeaTunnel也抽象了每种source所使用的参数，例如 `plugin_output` 参数，用于指定当前source生成的数据的名称，
-方便后续其他模块使用。
```

Comment:
In markdown syntax, blank lines are used to separate paragraphs. If there are no blank lines, all text is in the same paragraph. So I think we should keep it.